packages/video_player/video_player_platform_interface/lib/messages.dart

```diff
@@ -4,6 +4,8 @@
 import 'dart:async';
 import 'package:flutter/services.dart';
 
+// @dart = 2.8
```

Comment:
FYI @gaaclarke - any plans to add this to Pigeon or migrate it to null safety?

Reply:
I'll add the comment to pigeon for now and upgrade it in the future, thanks.